frontend/packages/console-app/src/actions/creators/deployment-factory.ts

```diff
@@ -61,6 +92,20 @@ export const DeploymentActionFactory: ResourceActionFactory = {
       verb: 'patch',
     },
   }),
+  RestartRollout: (kind: K8sKind, obj: K8sResourceKind): Action => ({
+    id: 'restart-rollout',
+    label: i18next.t('console-app~Restart rollout'),
+    cta: () => restartRollout(kind, obj).catch((err) => errorModal({ error: err.message })),
```

Comment:
There's a bug here:
https://user-images.githubusercontent.com/895728/175338045-a5d2b595-1610-4221-97a5-4c5c50d8953d.mov

Reply:
When I was testing it earlier I found that this action worked with certain deployments, but other deployments would cause the server to return a 422 error. How should I handle this on the frontend?